src/home/form/LoginForm.jsx

```diff
@@ -26,28 +28,35 @@ class Loginpage extends React.Component {
         this.setState({password: e.target.value});
     }
 
+    setErrorMessage() {
+        this.setState({errorMessage:'error-message'});
+    }
+
     sendToServer(e) {
         console.log('LoginForm is submitting the DATA MAN');
         e.preventDefault();
-
+        var self = this;
         axios.post('/login', this.state)
             .then(function (response) {
                 if (response.data !== '') {
                     document.location.href = '/dash';
                 } else {
                     console.log(response.data);
-                    // TODO show error message
+                    self.setErrorMessage();
```

Comment:
Yeah we should modify our backend to return more info about the error message. As @SrutiG suggested we can return a `json`. Then this can consume it and act accordingly.

Reply:
I can work on this, i totally agree